src/main/java/org/sonar/plugins/stash/issue/StashTask.java

```diff
@@ -0,0 +1,32 @@
+package org.sonar.plugins.stash.issue;
+
+public class StashTask {

```

Comment:
I would prefer to keep this simple class like this: it finally just a class to describe a Stash task.